desktop/renderer-app/src/utils/hooks/use-url-app-launcher.ts

```diff
@@ -38,19 +38,25 @@ export function useURLAppLauncher(): void {
     };
 
     useAutoRun(() => {
-        if (!urlProtocolStore.toJoinRoomUUID) {
-            return;
+        if (urlProtocolStore.toJoinRoomUUID || urlProtocolStore.toReplayRoom) {
```

Comment:
I think the condition here is `!urlProtocolStore.toJoinRoomUUID`, right ?

Reply:
Refactored.